### PR TITLE
Clean LHEInterface

### PR DESCRIPTION
**Pull request by mkirsano**

### CODE REVIEW

**Comment by govoni** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Comment by smuzaffar** [comment]
@mkirsano , looks like this broke few workflow which still access ExternalLHEProducer plugin
https://cms-sw.github.io/relvalLogDetail.html#slc6_amd64_gcc493;CMSSW_8_0_X_2016-01-28-2300

**Comment by covarell** [comment]
externalLHEProducer is used by most workflows in CMS, so we certainly have to reject this in this case. 

On the other hand, I do not see this plugin among the removed files

**Comment by covarell** [comment]
I see it now the problem is here:

https://github.com/cms-sw/cmssw/pull/13094/files#diff-e7ee1af94836eba237beb637a5794af7L8

Too many plugins removed in the BuildFile

File: GeneratorInterface/LHEInterface/plugins/BuildFile.xml [hunk 1]
```diff
@@ -5,11 +5,6 @@
 <use name="GeneratorInterface/LHEInterface"/>
 <use name="GeneratorInterface/Core"/>
 <use name="boost"/>
-<library name="GeneratorInterfaceLHEProducer" file="LHEProducer.cc LHEFilter.cc LHE2HepMCConverter.cc ExternalLHEProducer.cc ExternalLHEAsciiDumper.cc">
-	<use name="FWCore/Framework"/>
-        <use name="SimDataFormats/GeneratorProducts"/>
-	<flags EDM_PLUGIN="1"/>
-</library>
```
Comment (smuzaffar) [inline GeneratorInterface/LHEInterface/plugins/BuildFile.xml]: ExternalLHEProducer plugin is remove here.

**Comment by covarell** [comment]
-1

**Comment by smuzaffar** [comment]
this has been already merged. Should we revert it or you would like to just make a new PR with removed plugin added back?

**Comment by covarell** [comment]
@mkirsano or @smuzaffar please make a patch PR where only the files removed by this commit are also removed in the <lib> </lib> section of BuildFile

**Comment by davidlange6** [comment]
I'll just revert and @mkirsano can re-discuss with @covarell

**Pull request merged**